packages/ocean-core/src/components/_modal.scss

```diff
@@ -103,3 +103,10 @@ $max-widths: (
     }
   }
 }
+
+.ods-modal__content--lg {
+  @include media-breakpoint-down(md) {
+    border-radius: 0;
+    min-height: 100%;
```

Comment:
why not?
```suggestion
    height: 100%;
```

Reply:
`.ods-modal__content--lg` + `.ods-modal__content--blocked` in mobile I expected this behaviour as a lib consumer.

Reply:
cause i'd have to add `height: 100%` and alter the `max-height` to 100% as well, either way is fine imo

Reply:
@edu2711 what conclusion we have regarding this?